test/format/format.i

```diff
@@ -323,8 +383,6 @@ key_gen_insert(TABLE *table, WT_RAND_STATE *rnd, WT_ITEM *key, uint64_t keyno)
 static inline int
 lock_try_writelock(WT_SESSION *session, RWLOCK *lock)
 {
-    testutil_assert(LOCK_INITIALIZED(lock));
```

Comment:
Removing these seems a bit odd - is it just redundant or was there a better reason?

Reply:
The definition of this macro was removed too. See changes in `test/format/format.h`. What it was doing is just checking if the `lock_type` is not `LOCK_NONE`.

Reply:
The assert adds additional branching code into lock acquisition, that's going to slow things down in a performance path.